freqtrade/persistence/models.py

```diff
@@ -518,16 +560,20 @@ def calc_close_trade_value(self, rate: Optional[float] = None,
         if rate is None and not self.close_rate:
             return 0.0
 
-        sell_trade = Decimal(self.amount) * Decimal(rate or self.close_rate)  # type: ignore
-        fees = sell_trade * Decimal(fee or self.fee_close)
-        return float(sell_trade - fees)
+        close_trade = Decimal(self.amount) * Decimal(rate or self.close_rate)  # type: ignore
+        fees = close_trade * Decimal(fee or self.fee_close)
+        interest = ((self.interest_rate * Decimal(borrowed or self.borrowed)) * (datetime.utcnow() - self.open_date).days) or 0 #Interest/day * num of days
```

Comment:
i'd be careful with this - your assumption is that interrest is paid per day - but AFAIK, some exchanges calculate interrest per hour - but i guess we can leave that for later - when we get to the implementation of the different exchanges (basically - deferr that to when we need it).

Reply:
I'm assuming interest is calculated as an amount/day value, not sure if that's what you meant. I understand it's dangerous though and left a #TODO there